src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/reports/ReportsService.java

```diff
@@ -37,8 +49,24 @@ public ReportsService(
             .collect(toList());
     }
 
-    public List<ZipFileSummary> getZipFilesSummary(LocalDate date, String jurisdiction) {
-        throw new NotImplementedException("Not yet implemented");
+    public List<ZipFileSummaryResponse> getZipFilesSummary(LocalDate date, String jurisdiction) {
```

Comment:
It would be good to either add javadoc explaining that null or empty `jurisdiction` does not filter or add another method:
```
List<ZipFileSummaryResponse> getZipFilesSummary(LocalDate date)
```

Reply:
@rkondratowicz Added Javadoc